src/app/my-calc/my-calc.component.ts

```diff
@@ -35,25 +41,24 @@ export class MyCalcComponent implements OnInit {
 
   getValue (value:string) {
     
-    if (this.calcRezult != 0 && (value ===  "+" || value === "-" || value === "*" || value === "/")){
+    if (this.calcRezult != 0 && value === checkEqual(value)){
```

Comment:
Pokusaj kod imena i variabli malo konkretnija imena dodjelivati. Ovo `checkEqual(value)` je previs generalno i nako par sedmica sam neces znati sta funkcija radi. 

Pogledaj https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c